tests/baselines/reference/templateStringInYieldKeyword.js

```diff
@@ -6,7 +6,7 @@ function* gen() {
 
 
 //// [templateStringInYieldKeyword.js]
-function gen() {
+function* gen() {
     // Once this is supported, the inner expression does not need to be parenthesized.
     var x = yield "abc" + x + "def";

```

Comment:
We can consider this for later, for now I'm just working on wrapping up our existing test failures with the transformations.